dataland-frontend/src/components/general/DataPointDataTable.vue

```diff
@@ -5,7 +5,7 @@
         <tbody class="p-datatable-body">
           <tr
             v-if="
-              dialogData.dataPointDisplay.value && dialogData.dataPointDisplay.value != ONLY_AUXILIARY_DATA_PROVIDED()
+              dialogData.dataPointDisplay.value && dialogData.dataPointDisplay.value != ONLY_AUXILIARY_DATA_PROVIDED
             "
```

Comment:
This is just a refactoring?

Reply:
Ja. Es macht keinen Sinn unten eine Methode zu definieren, die dieses Objekt einfach zurückgibt.
Es ist sinnvoller, das so zu machen, wie ich es hier gemacht habe (deutlich weniger lines of code).